### ISSUE TITLE
Baton/Sarcastic Discard Ejection

### ISSUE THREAD

**Issue by pianoblook** [Zamiell/hanabi-conventions#301]
Baton Discards are noted to be almost exactly like _Gentlemen's Discards_ except for currently unplayable cards. The doc says, "Note that it is illegal to perform a Layered Baton Discard; the card must be exactly in Finesse Position."

But what if it could be done, if it's promising something extra & precise? This would function exactly like the [Gentleman's Discard Ejection](https://github.com/Zamiell/hanabi-conventions/blob/master/Reference.md#the-gentlemans-discard-ejection) (and GDD by extension, if/when that's accepted).

Example from a recent game below. Assuming pianoblook fully knows he has pink 2 and discards it, the only way for things to not go terribly wrong would be if there was a clear and valuable way for piper to know which position his Pink 2 is. (Assume it's 100% known here, even though it's pink - or replace pink with a fully-clued purple 2 for simplicity).

If piper was currently unclued, he would assume this was a _Baton Discard_ and mark slot 1 as the 2. In a situation like this, where he has one or more previously clued pink cards, he's going to mark one of them as the 2.

If melwen (the immediate next player) blindplays her Second Finesse position, though, this would signal to Piper exactly what is going on.

<img width="1280" alt="Baton Discard Ejection" src="https://user-images.githubusercontent.com/32303453/85954441-682ec900-b945-11ea-8bad-4ff9c8ffdbc4.png">

**Comment by pianoblook** [comment]
And yes, I see that piano could just get the brown 1 with a 5-pull on melwen. In this case it would also start to unpleasantly clog piper's hand so I didn't actually go for it. I had not seen anything like this  before though and thought it was neat.

**Comment by Zamiell** [comment]
i think that this probably flows from the existing GDE convention with any explicit pre-coordination needed

**Comment by Zamiell** [comment]
blocked by #302

**Comment by Zamiell** [comment]
closing this in favor of 302
